sympy/utilities/lambdify.py

```diff
@@ -386,6 +386,14 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     namespace.update({'builtins':builtins, 'range':range})
 
     func = eval(lstr, namespace)
+    # For numpy lambdify, wrap all input arguments in arrays.
+    # This is a fix for gh-11306.
+    if module_provided and _module_present('numpy',namespaces):
```

Comment:
I wrote it that way to be consistent with the appearance of the same function call earlier in the module. I do not know the SymPy policy on coding style.